Examples/Python/ellipsoid.py

```diff
@@ -84,13 +84,14 @@ def Run_Pipeline(args):
         paddedFiles = applyPadding(groomDir + "padded/segmentations", centeredFiles, 10)
 
         """Apply center of mass alignment"""
-        comFiles = applyCOMAlignment(groomDir + "com_aligned/segmentations", centeredFiles, None)
+        comFiles = applyCOMAlignment(groomDir + "com_aligned/segmentations", paddedFiles, None)
```

Comment:
I think we don't need padding as part of the workflow before alignment. we can pad when we do cropping. Please see the grooming notebook. @jadie1

Reply:
Fixed b3d9b55